### ISSUE TITLE
SEPA doesn't support subscription products in split UPE

### ISSUE THREAD

**Issue by timur27** [Automattic/woocommerce-payments#5517]
Split UPE doesn't support checking out subscription products with SEPA.

With an old UPE in `develop`:

* Users can use SEPA to buy a subscription product
* Users can re-use SEPA, which are already on the saved payment methods list
* Users cannot save SEPA they use when checking out a subscription.

On the split UPE, however:
* Users cannot use SEPA to buy a subscription product
* Users see SEPA methods that were saved previously, but when using them, there’s an error

Acceptance criteria: 
* Users can use SEPA to buy a subscription product
* Users can re-use SEPA, which are already on the saved payment methods list
* Users cannot save SEPA they use when checking out a subscription.

**Comment by timur27** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Heisenberg-63586f2ec60359001310fa6a/issues/Automattic/woocommerce-payments/5517?planning-poker) with Zenhub @FangedParakeet @mdmoore

**Comment by frosso** [comment]
Moving to ToDo - de-prioritized SEPA compatibility for the Split UPE.

**Comment by zmaglica** [comment]
@frosso Are you or perhaps your team planning to work on this issue? The reason why I am asking is that I have to triage it and put it in prioritization column, and if you are planning to work on it, I can move it to different queue.

**Comment by frosso** [comment]
@zmaglica yep, we're planning to work on this as a low priority item.

**Comment by thenbrent** [comment]
> Users cannot save SEPA they use when checking out a subscription.

@timur27 are you aware of this ever working? I'm investigating and AFAICS, SEPA has never worked for processing of automatic recurring payments even though it was presented as an option during checkout.

**Comment by timur27** [comment]
Hey @thenbrent - sorry for the late response! I think there are two things and I might have provided a bit unclear description. 

1️⃣ 
> Users cannot save SEPA they use when checking out a subscription.

What I had in mind here was that there is no _Save payment information to my account for future purchases._ checkbox provided for users, when they check out using SEPA with a subscription. This is expected, and I probably need to rephrase this point in the description to make it more clear. 

------------

2️⃣ 
> SEPA has never worked for processing of automatic recurring payments even though it was presented as an option during checkout

I've re-tested it now to refresh my memory. SEPA can be used with subscriptions, and when I processed a subscription renewal for a product bought with SEPA, the process went smoothly. SEPA is also saved in _My account -> Payment methods_ page, even though there's no checkbox to save it on the checkout page. In other words, it behaves the same way as a credit card. 

Please let me know if this all makes sense or if I misunderstood your initial question! Thanks.

**Comment by thenbrent** [comment]
That's a great help, thanks @timur27.

The original issue description also mentions that on the split UPE, users cannot use SEPA to buy a subscription product. 

Is the Split UPE only available to newer merchants? i.e. those that installed WC Pay after a certain date. If so, what is the that cut-off date?

I'm trying to determine if the split UPE incompatibility could be the root cause of #3776 and some of the other issues we're seeing reported around SEPA usage with subscriptions (see P2 thread here with more details pchH0V-18S-p2#comment-1311).

> I've re-tested it now to refresh my memory. SEPA can be used with subscriptions, and when I processed a subscription renewal for a product bought with SEPA, the process went smoothly.

Just to clarify, that testing is with the old UPE in `develop`, not the split UPE? (Because this bug hasn't been fixed.)

cc: @csmcneill

**Comment by timur27** [comment]
Thanks for the context, @thenbrent! Here are two main points that came to my mind: 

1. Split UPE was introduced in WCPay `5.5.0` from 2023-02-22, whereas [the bug](https://github.com/Automattic/woocommerce-payments/issues/3776) appears starting from Feb 2022
2. Split UPE has bigger incompatibility (which https://github.com/Automattic/woocommerce-payments/issues/5517 should solve) at the moment with the SEPA payment method, which is why [the fallback to the legacy UPE was introduced](https://github.com/Automattic/woocommerce-payments/pull/5552) when users have split UPE and SEPA enabled. Essentially, this implies that when someone has split UPE enabled and activates SEPA for their Stripe account, their configuration will automatically switch to the legacy UPE.

Having that said, split UPE is unrelated to the problem described in https://github.com/Automattic/woocommerce-payments/issues/3776, in my opinion. Please let me know if I missed anything or if anything isn't clear enough and I'm happy to align further! Thank you.

**Comment by frosso** [comment]
Looks like in https://github.com/Automattic/woocommerce-payments/issues/3776 there's been a decision to ensure SEPA shouldn't support subscriptions/recurring payments, so we can ignore this for Split UPE/DI Split UPE.

**Comment by frosso** [comment]
re-opening 😞 
We'll need a decision here before we can fully close this: paJDYF-9Ud-p2

**Comment by frosso** [comment]
SEPA support for subscription payments will come to a later date, the solution will be revisited once SEPA becomes widely available.

It can take up to 14 business days to receive notification on the success or failure of a payment with SEPA. Woo Subscriptions' [renewal process](https://href.li/?https://woocommerce.com/document/subscriptions/renewal-process/#the-renewal-process) is designed with the assumption that payment notification is near immediate (or at least within 24 hours).

For context on this decision: paJDYF-9Ud-p2#comment-20540

Closing again.
